tensorflow/python/framework/byte_swap_tensor.py

```diff
@@ -54,12 +54,12 @@ def byte_swap_tensor_content(tensor, from_endiness, to_endiness):
       tensor_size = 1
       for sz in tshape:
         if sz.size != 0:
-          tensor_size = tensor_size * sz.size
-      chunksize = int(len(tensor_bytes) / tensor_size)
+          tensor_size *= sz.size
+      chunksize = len(tensor_bytes) // tensor_size
       # Split tensor_data into chunks for byte swapping.
       to_swap = [
           tensor_bytes[i : i + chunksize]
-          for i in range(0, len(tensor_bytes), chunksize)
+          for i in range(len(tensor_bytes), chunksize)
```

Comment:
Actually, this is wrong. You have `range` with 2 arguments here.

Reply:
Thank you for your reply !!
I will fix it